### PR TITLE
Don't use git "-C" flag

### PR DESCRIPTION
**Pull request by bibliotechy**
Git -C was introduced in git 1.8.5, but the latets on current CentOS (and similar) is 1.8.3.1

Replaces calls to -C with `cd` into that directory and then `cd` back to
original directory

### CODE REVIEW

File: update_build.sh [hunk 1]
```diff
@@ -9,13 +9,15 @@ docker-compose down
 # init Combine app submodule and use localsettings docker template
 git submodule init
 git submodule update
-git -C combine/combine fetch
-git -C combine/combine checkout $COMBINE_BRANCH
-git -C combine/combine pull
+cd combine/combine
+git fetch
+git $COMBINE_BRANCH
```
Comment (antmoth) [inline update_build.sh]:
```suggestion
git checkout $COMBINE_BRANCH
```

File: first_build.sh [hunk 2]
```diff
@@ -6,9 +6,11 @@ source .env
...
 git submodule init
 git submodule update
-git -C combine/combine fetch
-git -C combine/combine checkout $COMBINE_BRANCH
-git -C combine/combine pull
+cd combine/combine
+git fetch
+git checkout $COMBINE_BRANCH
+git  pull
+cd ../../
```
Comment (antmoth) [inline first_build.sh:14]:
```suggestion
```

File: first_build.sh [hunk 3]
```diff
@@ -6,9 +6,11 @@ source .env
...
 cp ./combine/combine/combine/localsettings.py.docker ./combine/combine/combine/localsettings.py
```
Comment (antmoth) [inline first_build.sh]:
```suggestion
cp ./combine/localsettings.py.docker ./combine/localsettings.py
```

File: first_build.sh [hunk 4]
```diff
@@ -6,9 +6,11 @@ source .env
...
```
Comment (antmoth) [inline first_build.sh]:
```suggestion
cd ../../
```

**Comment by bibliotechy** [comment]
@estelendur thanks for the suggestions, and catching my overzealous deletion. 

Applied.

**Pull request merged**